tools/airgap-install/get-image-list

```diff
@@ -112,19 +113,27 @@ echo ""
 echo "###################################"
 echo "# Getting Gloo Mesh images        #"
 echo "###################################"
-wget -q https://storage.googleapis.com/gloo-mesh-enterprise/gloo-mesh-enterprise/gloo-mesh-enterprise-${VERSION}.tgz
-tar zxf gloo-mesh-enterprise-${VERSION}.tgz
-find gloo-mesh-enterprise -name "values.yaml" | while read file; do
-    cat $file | yq eval $JSON_FLAG | jq -r '.. | .image? | select(. != null) | (if .registry then (if .registry == "docker.io" then "docker.io/library" else .registry end) + "/" else "" end) + .repository? + ":" + (.tag? | tostring)'
-done | sort -u | tee -a .images.out
+if [[ $VERSION == 2.0* || $VERSION == 2.1* || $VERSION == 2.2* || $VERSION == 2.3* ]]; then
+    wget -q https://storage.googleapis.com/gloo-mesh-enterprise/gloo-mesh-enterprise/gloo-mesh-enterprise-${VERSION}.tgz
+    tar zxf gloo-mesh-enterprise-${VERSION}.tgz
+    find gloo-mesh-enterprise -name "values.yaml" | while read file; do
+        cat $file | yq eval $JSON_FLAG | jq -r '.. | .image? | select(. != null) | (if .registry then (if .registry == "docker.io" then "docker.io/library" else .registry end) + "/" else "" end) + .repository? + ":" + (.tag? | tostring)'
+    done | sort -u | tee -a .images.out
+else
+    wget -q https://storage.googleapis.com/gloo-platform/helm-charts/gloo-platform-${VERSION}.tgz
+    tar zxf gloo-platform-${VERSION}.tgz
+    find gloo-platform -name "values.yaml" | while read file; do
+        cat $file | yq eval $JSON_FLAG | jq -r '.. | .image? | select(. != null) | (if .registry then (if .registry == "docker.io" then "docker.io/library" else .registry end) + "/" else "" end) + .repository? + ":" + (.tag? | tostring)'
+    done | sort -u | tee -a .images.out
+fi
 
 
 echo ""
 echo "###################################"
 echo "# Getting Solo istio images       #"
 echo "###################################"
```

Comment:
Small suggestion, but we've been rephrasing how we talk about Istio similar to the following:
```suggestion
echo "#######################################"
echo "# Getting Solo distributions of Istio #"
echo "#######################################"
```